### ISSUE TITLE
About the metadata display problem of more items.

### ISSUE THREAD

**Issue by apachezy** [BentoBoxWorld/Challenges#277]
#### Is your feature request related to a problem?

Now that the metadata of more items is displayed, the information display frame has become very large.

#### Describe the solution you'd like us to implement.

Use chest GUI to display required items and reward items.

#### Describe alternatives you've considered.

Add the option of whether to display metadata(This is not the best solution).

### Additional context
Already like this:
![Already like this](https://user-images.githubusercontent.com/50116371/111194066-c4ca3500-85f5-11eb-983e-bd6443a93212.png)

**Comment by PancakeCatsss** [comment]
Ah yes, was just about to make a feature request for this, a GUI to show items required would be great, we currently have a challenge that requires the player to get 32 of every copper block which results in a very large lore and players can't see all the items needed. I think overall a GUI to show items would work out better in the long run and be easier for players to view them. 

Hope this is added but overall great addon.

**Comment by BONNe** [comment]
Yes, I agree, but creating GUI like that takes time.

I am planning to add one eventually ... not in 1.0 release.

**Comment by PancakeCatsss** [comment]
Not to worry, quality takes time :)

**Comment by tastybento** [comment]
Fair point — for item-heavy challenges the lore turns into a wall of text, and with dozens of distinct required items it's genuinely unreadable. There's a per-challenge option to hide the reward/requirement text, but no visual alternative.

What I'd do: a simple read-only preview panel that shows the required items (and reward items) as actual ItemStacks in a grid, opened from a button on the challenge when the item list is long. Template-driven like the other panels, no new config beyond maybe a threshold.

Marking P2 — a UX improvement rather than a bug, but self-contained panel work.
